src/password.h

```diff
@@ -0,0 +1,10 @@
+#ifndef PASSWORD_H
+#define PASSWORD_H
+
+#include <cstring>
```

Comment:
Header files should only include the files they *themselves* need. Things that the corresponding `.cpp` file needs should be included in that `.cpp`.